rpcapd/daemon.c

```diff
@@ -59,6 +55,10 @@ typedef int ssize_t;
 #include <shadow.h>		// for password management
 #endif
 
+#if defined(_WIN32) && !defined(_SSIZE_T_DEFINED)
```

Comment:
Again, no longer necessary, as `daemon.c` no longer uses `ssize_t`.

Reply:
> Again, no longer necessary, as daemon.c no longer uses ssize_t.

Since when? It seemed necessary when pushed the PR and
I can't really check right now.

Reply:
Since de7aab10f11dd9b64d55afffaeb40e264370ce8d.